hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/logs/container/ListContainers.java

```diff
@@ -57,7 +65,11 @@ public Void call() throws Exception {
 
     ContainerDatanodeDatabase cdd = new ContainerDatanodeDatabase(dbPath.toString());
 
-    cdd.listContainersByState(state.name(), listOptions.getLimit());
+    if (exclusiveOptions.duplicateOpen) {
+      cdd.findDuplicateOpenContainer();
+    } else if (exclusiveOptions.state != null) {
+      cdd.listContainersByState(exclusiveOptions.state.name(), listOptions.getLimit());
+    }
```

Comment:
`ExclusiveOptions` only forces choice between `--state` and `--duplicate-open`.  But neither is `--limit` applicable to `findDuplicateOpenContainer()`.  I think this should be a separate subcommand.  Please leave `ListContainers` unchanged.